resources/measures/upgrade_hvac_enable_ideal_air_loads/measure.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<measure>
+  <schema_version>3.1</schema_version>
+  <name>upgrade_hvac_enable_ideal_air_loads</name>
+  <uid>f70d4a3d-8b93-4fef-b679-858fe1e05acc</uid>
+  <version_id>a0e3eacb-5341-4b71-9a05-fa253a922db1</version_id>
+  <version_modified>2024-07-23T14:05:58Z</version_modified>
+  <xml_checksum>4435983D</xml_checksum>
+  <class_name>UpgradeHvacEnableIdealAirLoads</class_name>
+  <display_name>UpgradeHvacEnableIdealAirLoads</display_name>
+  <description>This will enable Ideal Air Loads for all thermal zones in the model. This will result in disconnecting any pre-existing HVAC systems from the zones.</description>
+  <modeler_description>This doesn't clean up any air or plant loops that are left. It was developed to use on models without HVAC. If you use it in another way you may have to clean up remaining objects prior to running a simulation.</modeler_description>
```

Comment:
Don't think this has been updated, since there's no `modelerDescription` in the measure.rb, and besides the measure calls `remove_hvac`. It would be nice to have an update measure description.